lesson09/server9/image_scratch/restapi/main.go

```diff
@@ -0,0 +1,78 @@
+package main
```

Comment:
This is not what you were expected to do in this task. You need to create a package, that will interact with a database: create new entries and fetch existing ones. What you have here is just a REST api that stores everything in memory. 
You can make a REST api, that displays all that in the browser, but as it was explained in the lecture, you don't have to. Writing a package, that works with Postgres and showing it's usage is sufficient

Aslo, calling your entries "tokens" is not good. It doesn't reflect he main purpose of what the variables hold. You need to get used to name everything appropriately, as it will make code cleaner and easier to follow for everyone, including yourself. It would be better to use `contacts` in these variables and functions associated with their interaction